### PR TITLE
add setEnvironment and setCallbackUrl

### PR DESCRIPTION
**Pull request by jumaallan**
Story: https://app.shortcut.com/smileid/story/13086/add-missing-interfaces-on-react-native-sdk

## Summary

Added missing setEnvironment and setCallbackUrl interfaces

## Known Issues

Any shortcomings in your work. This may include corner cases not correctly handled or issues related
to but not within the scope of your PR. Design compromises should be discussed here if they were not
already discussed above.

## Test Instructions

Concise test instructions on how to verify that your feature works as intended. This should include
changes to the development environment (if applicable) and all commands needed to run your work.

## Screenshot

If applicable (e.g. UI changes), add screenshots to help explain your work.

### CODE REVIEW

File: android/src/main/java/com/smileidentity/react/SmileIdModule.kt [hunk 1]
```diff
@@ -55,6 +56,18 @@ class SmileIdModule internal constructor(context: ReactApplicationContext) :
...
+  }
+
+  @ReactMethod
+  override fun setCallbackUrl(callbackUrl: String, promise: Promise) {
+    SmileID.setCallbackUrl(callbackUrl = URL(callbackUrl))
+    promise.resolve(null)
+  }
+
   @ReactMethod
   override fun disableCrashReporting(promise: Promise) {
```
[the remark this answers is not in the document]
Comment (jumaallan) [reply android/src/main/java/com/smileidentity/react/SmileIdModule.kt:72]: undone the change
